<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1219</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Korea Invest API2</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="accountTableView">
    <property name="geometry">
     <rect>
      <x>353</x>
      <y>35</y>
      <width>861</width>
      <height>308</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>90</number>
    </attribute>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>355</x>
      <y>20</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>계좌 정보</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>40</y>
      <width>307</width>
      <height>140</height>
     </rect>
    </property>
    <property name="title">
     <string>매수 조건</string>
    </property>
    <widget class="QComboBox" name="buyRSITypeComboBox">
     <property name="geometry">
      <rect>
       <x>168</x>
       <y>98</y>
       <width>75</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>상향돌파</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>하향돌파</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>이상</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>이하</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>23</x>
       <y>62</y>
       <width>142</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>MACD가 시그널 라인을</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>100</y>
       <width>62</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>RSI 값이</string>
     </property>
    </widget>
    <widget class="QComboBox" name="buyMACDTypeComboBox">
     <property name="geometry">
      <rect>
       <x>167</x>
       <y>59</y>
       <width>75</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>상향돌파</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>하향돌파</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>이상</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>이하</string>
      </property>
     </item>
    </widget>
    <widget class="QSpinBox" name="buyRSIValueSpinBox">
     <property name="geometry">
      <rect>
       <x>94</x>
       <y>97</y>
       <width>55</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>24</x>
       <y>27</y>
       <width>88</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>매수당 금액:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="buyAmountLineEdit">
     <property name="geometry">
      <rect>
       <x>109</x>
       <y>25</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>100000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>193</y>
      <width>307</width>
      <height>124</height>
     </rect>
    </property>
    <property name="title">
     <string>매도 조건</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>23</x>
       <y>37</y>
       <width>142</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>MACD가 시그널 라인을</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>75</y>
       <width>62</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>RSI 값이</string>
     </property>
    </widget>
    <widget class="QComboBox" name="sellRSITypeComboBox">
     <property name="geometry">
      <rect>
       <x>165</x>
       <y>73</y>
       <width>75</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>상향돌파</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>하향돌파</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>이상</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>이하</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="sellMACDTypeComboBox">
     <property name="geometry">
      <rect>
       <x>165</x>
       <y>34</y>
       <width>75</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>상향돌파</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>하향돌파</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>이상</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>이하</string>
      </property>
     </item>
    </widget>
    <widget class="QSpinBox" name="sellRSIValueSpinBox">
     <property name="geometry">
      <rect>
       <x>92</x>
       <y>72</y>
       <width>55</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
   </widget>
   <widget class="QTableView" name="watchListTableView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>363</y>
      <width>1191</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>343</y>
      <width>70</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Watchlist</string>
    </property>
   </widget>
   <widget class="QLabel" name="domesticCurrentBalanceLabel">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>14</y>
      <width>242</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>현재 평가 잔고:</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
